### PR TITLE
Fix invalid unit test naming causes unit test not to be performed

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:
Fix invalid unit test naming causes unit test not to be performed. Unit test function names must begin with `Test` to be executed
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
/sig testing

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wgliang** [comment]
/cc @mwielgus @DirectXMan12

PTAL,thanks.

File: pkg/controller/podautoscaler/legacy_horizontal_test.go [hunk 1]
```diff
@@ -466,22 +466,26 @@ func (tc *legacyTestCase) runTest(t *testing.T) {
...
-			case "SuccessfulRescale":
-				assert.Equal(t, fmt.Sprintf("New size: %d; reason: cpu resource utilization (percentage of request) above target", tc.desiredReplicas), obj.Message)
-			case "DesiredReplicasComputed":
-				assert.Equal(t, fmt.Sprintf(
-					"Computed the desired num of replicas: %d (avgCPUutil: %d, current replicas: %d)",
-					tc.desiredReplicas,
-					(int64(tc.reportedLevels[0])*100)/tc.reportedCPURequests[0].MilliValue(), tc.initialReplicas), obj.Message)
-			default:
-				assert.False(t, true, fmt.Sprintf("Unexpected event: %s / %s", obj.Reason, obj.Message))
+		switch action.(type) {
```
Comment (DirectXMan12) [inline pkg/controller/podautoscaler/legacy_horizontal_test.go]: what's up with the seemingly unrelated change?  Can you put this in a separate commit with a description?

**Review by DirectXMan12 — CHANGES_REQUESTED** [review]
minor nit inline, otherwise looks fine.

**Comment by DirectXMan12** [comment]
if you're still interested in this PR, please respond to the comment

**Pull request closed without merging**